gm4_desire_lines/data/gm4_desire_lines/functions/player.mcfunction

```diff
@@ -7,4 +7,5 @@ execute if score @s gm4_desire_lines matches 30.. run scoreboard players set @s
 execute if score @s gm4_desire_lines matches 1.. run scoreboard players remove @s gm4_desire_lines 20
 
 function #gm4_desire_lines:expansion
+scoreboard players set @s[nbt={ActiveEffects:[{id:28}]}] gm4_desire_lines 0
```

Comment:
Shouldn't this be `Id`?

Reply:
Ideally it should use a predicate to detect the effect. I believe this should do it.
```json
{
  "condition": "minecraft:entity_properties",
  "entity": "this",
  "predicate": {
    "effects": {
      "slow_falling": {}
    }
  }
}
```

Reply:
:'D Asks in a voice call that Misode is in whether I should use a predicate or NBT and told either is fine, use nbt 🗡️